manifests/0000_90_cluster-svcat-controller-manager-operator_02_servicemonitor.yaml

```diff
@@ -24,3 +24,20 @@ spec:
   selector:
     matchLabels:
       app: openshift-service-catalog-controller-manager-operator
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: openshift-service-catalog-controller-manager-operator
+  namespace: openshift-service-catalog-controller-manager-operator
+spec:
+  groups:
+    - name: svcat-controller-manager-enabled
+      rules:
+        - alert: ServiceCatalogControllerManagerEnabled
+          annotations:
+            summary: Indicates whether Service Catalog is enabled
```

Comment:
Probably doesn't matter since the alert name is different, but should we distinguish the summary from the [API server summary](https://github.com/openshift/cluster-svcat-apiserver-operator/pull/66/files#diff-871d20bc21cb61a48dd6b88dd4fe6a2dR40)?

Reply:
Updated